app/policies/drawless_group_policy.rb

```diff
@@ -2,4 +2,7 @@
 #
 # Policy for permissions on special (non-draw) housing groups
 class DrawlessGroupPolicy < ApplicationPolicy
+  def select_suite?
+    user.admin? && record.full?
```

Comment:
possibly this also means that we should default admin created groups to be always locked